src/content/reference/docker-compose.mdx

```diff
@@ -563,3 +563,34 @@ endpoints:
       service: backend
       port: 8080
 ```
+
+### update (annotation, optional)
+
+You can set the update policy of a service by setting the annotation `dev.okteto.com/update`. 
+
+Depending on the kind of resource the acceptance values are different:
+- rolling: Will deploy a service with 0 downtime. Allowed on deployments and statefulsets.
```

Comment:
what do we mean by 0 downtime? can we guarantee that?
```suggestion
- rolling: Will deploy a service with zero downtime. Only allowed for deployments and stateful sets.
```

Reply:
what happens if I set this for something that's not a deployment or a stateful set?

Reply:
> what happens if I set this for something that's not a deployment or a stateful set?

It will work the same as it was working previously. This change is only for sfs and deployments